src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -20,6 +22,7 @@ import * as admin from "firebase-admin";
 import * as bodyParser from "body-parser";
 import { pathToFileURL, URL } from "url";
 import * as _ from "lodash";
+import { CloudEvent } from "firebase-functions/lib/v2";
```

Comment:
1. This will crash in Node 12 and above
2. This makes me very nervous since `firebase-functions` is only a dev dependency. I wouldn't mind a comment warning future devs to only import interfaces because the SDK is only a dev dependency.

Reply:
Great points. I don't think correctly typing the function that received `CloudEvent` is that useful anyway - removing this import.